session/session_other.go

```diff
@@ -6,17 +6,13 @@
 
 package session
 
-import (
-	"fmt"
-	"os"
-)
-
 // Namespace assembles a NameSpace for this cpud, iff CPU_NAMESPACE
 // is set.
 // CPU_NAMESPACE can be the empty string.
 // It also requires that CPU_NONCE exist.
 func (s *Session) Namespace() error {
-	return fmt.Errorf("CPUD: 9p mounts are only valid on Linux:%w", os.ErrNotExist)
+	//return fmt.Errorf("CPUD: 9p mounts are only valid on Linux:%w", os.ErrNotExist)
```

Comment:
I'm unclear on what the reason for this is. Why is it not an error? Why is the error commented out, if nil is OK?